vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/GridMultiSelectionColumnPageIT.java

```diff
@@ -76,6 +103,11 @@ public void noSelectOnRowItemClick() {
                 .get().click();
         Assert.assertEquals("No selection event should be fired", "",
                 findElement(By.id("selected-item-count")).getText());
+
+        // Test switch selection mode from Multi to single mode before adding the grid to DOM
+        // By checking the vaadin-grid-cell-content
+        WebElement gridSelectionMode = findElement(By.id("in-testing-multi-selection-mode-grid-single"));
+        Assert.assertEquals(0, gridSelectionMode.findElements(By.tagName("vaadin-checkbox")).size());
```

Comment:
`Assert.assertTrue(gridSelectionMode.findElements(By.tagName("vaadin-checkbox")).isEmpty())` would be more readable.

Reply:
done